src/main/java/org/cardanofoundation/job/schedules/BlockStatisticsSchedule.java

```diff
@@ -0,0 +1,99 @@
+package org.cardanofoundation.job.schedules;
+
+import java.util.List;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.RestTemplate;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import org.cardanofoundation.explorer.common.entity.explorer.BlockStatisticsDaily;
+import org.cardanofoundation.explorer.common.entity.explorer.BlockStatisticsPerEpoch;
+import org.cardanofoundation.job.repository.explorer.jooq.JOOQBlockStatisticsDailyRepository;
+import org.cardanofoundation.job.repository.explorer.jooq.JOOQBlockStatisticsPerEpochRepository;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+@ConditionalOnProperty(
+    value = "jobs.block-statistics.enabled",
+    matchIfMissing = true,
+    havingValue = "true")
+public class BlockStatisticsSchedule {
+  private final RestTemplate restTemplate = new RestTemplate();
+
+  @Value("${jobs.block-statistics.daily-url}")
+  private String apiBlockStatisticsDailyUrl;
+
+  @Value("${jobs.block-statistics.per-epoch-url}")
+  private String apiBlockStatisticsPerEpochUrl;
+
+  private final JOOQBlockStatisticsDailyRepository jooqBlockStatisticsDailyRepository;
+
+  private final JOOQBlockStatisticsPerEpochRepository jooqBlockStatisticsPerEpochRepository;
+
+  @Scheduled(cron = "0 0 0 * * *", zone = "UTC")
```

Comment:
Why is this schedule fix and not configurable?

Reply:
https://cardanofoundation.atlassian.net/browse/MET-2247
This scheduler required daily refesh (midnight UTC)